auction/README.md

```diff
@@ -20,45 +20,146 @@ We make following assumptions about the auction:
 Under this model, we require that:
 - Bidders do not learn any information about the value of other bids.
 
-### Auction Flow
+## Auction Flow
+
 The auction is conducted in a series of stages.
 - **Bidding**: In the bidding stage, bidders submit bids to the auctioneer. They do so by invoking the `place_bid` function.
 - **Resolution**:  In the resolution stage, the auctioneer resolves the bids in the order they were received. The auctioneer does so by invoking the `resolve` function. The resolution process produces a single winning bid.
 - **Finishing**: In this stage, the auctioneer finishes the auction by invoking the `finish` function. This function returns the winning bid to the bidder, which the bidder can then use to claim the item.
 
-
 ## Language Features and Concepts
 - `record` declarations
 - `assert_eq`
 - record ownership
 
-## Running the Program
+## Setup
+
+### 1. Install Leo
+Follow the [Leo Installation Instructions](./01_installation.md).
+
+### 2. Download the auction example code
+Clone the source code for the auction example from [Github](https://github.com/AleoHQ/workshop/tree/master/auction).
+
+## How to Run
+
+The example should work out of the box. We recommend running the entire script, and then creating your own accounts to replicate the functionality. Run the entire script:
```

Comment:
```suggestion
This auction program can be run using the following bash script.
Locally, it will execute Leo program functions to conduct, bid, and close a three party auction.
```
We should explain what the run script does

Simplify